app/src/main/java/com/sdpteam/connectout/mapList/MapListModelManager.java

```diff
@@ -0,0 +1,17 @@
+package com.sdpteam.connectout.mapList;
+
+import androidx.lifecycle.MutableLiveData;
+
+import com.sdpteam.connectout.event.Event;
+
+import java.util.List;
+
+public interface MapListModelManager {
```

Comment:
Rename to `EventRepository` and move it to the event folder
Don't forget as said before to change to CompletableFuture